### PR TITLE
allows user to add journal entry to database

### PR DESCRIPTION
**Pull request by wes-mitchell**
Completed as a team.

### CODE REVIEW

**Review by Nayhee — DISMISSED** [review]
Wow those changes look good! Whew!

File: TabloidCLI/Repositories/JournalRepository.cs [hunk 1]
```diff
@@ -4,10 +4,48 @@
...
+            }
+        }
+        public List<Journal> GetAll()
+        {
+            List<Journal> list = new List<Journal>();
+            return list;
+        }
+        public Journal Get(int id)
+        {
+            Journal test = new Journal();
```
Comment (DakotaLambert) [inline TabloidCLI/Repositories/JournalRepository.cs:39]: I don't think a "test" variable is very clear. 

I would rename these to something a little more descriptive  😄 @wes-mitchell

Comment (wes-mitchell) [reply TabloidCLI/Repositories/JournalRepository.cs]: You right, I changed those bad boys.

**Review by Nayhee — APPROVED** [review]
changes look great. Solid work.

**Review by DakotaLambert — APPROVED** [review]
Very nice, thank you! 🧙🏻

**Pull request merged**